app/controllers/notes_controller.rb

```diff
@@ -90,8 +90,10 @@ def raw
 
   def show
     if params[:author] && params[:date]
-      @node = DrupalNode.where(path: "/notes/#{params[:author]}/#{params[:date]}/#{params[:id]}").first
+      finder = "#{params[:author]} #{params[:date]} #{params[:id]}".parameterize
```

Comment:
@jywarren This keeps the url format intact. I am just creating a finder string from the parameters that finds the appropriate node using the friendly_id slug. Thus we can use best of both worlds! Not modifying the url format as well as using the friendly_id for old url redirects. while updating the url as the title changes.